Tests/scripts/update_id_set.py

```diff
@@ -503,10 +521,12 @@ def update_id_set():
 
     if added_files or modified_files:
         new_ids_dict = OrderedDict()
-        new_ids_dict['scripts'] = script_set
-        new_ids_dict['playbooks'] = playbook_set
-        new_ids_dict['integrations'] = integration_set
-        new_ids_dict['TestPlaybooks'] = test_playbook_set
+        # we sort each time the whole set in case someone manually changed something
+        # it shouldn't take too much time
+        new_ids_dict['scripts'] = sort(script_set)
```

Comment:
These sorts should take a lot, no? you are sorting a file of 15K lines..

Reply:
it actually works very fast because we have it all in the memory and we sort in total ~1000 items in array

Reply:
milliseconds